ansible_collections/arista/avd/docs/release-notes/5.x.x.md

```diff
@@ -41,6 +41,58 @@ for details.
 
 TODO: add level4 sections for breaking changes outlining the change and referring to porting guide for migration.
 
+### Breaking or behavioral changes in eos_validate_state
+
+Starting AVD 5.0.0, `eos_validate_state` uses ANTA as backend, which will change the default behavior (same as setting `use_anta: true` in older versions).
+If you have not been using `use_anta: true` in latest versions of AVD 4.x.x, you should expect faster execution, and if not please report on the GitHub [discussions board](https://github.com/aristanetworks/avd/discussions)
```

Comment:
```suggestion
If you have not been using `use_anta: true` in the latest versions of AVD 4.x.x, please go through the following sections for details on the changes.

Please report any issues on the GitHub [discussions board](https://github.com/aristanetworks/avd/discussions).
```

Reply:
this removes the point of execution being faster though